experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -44,20 +44,25 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
 // ExpanderReconciler reconciles a expander object
 type ExpanderReconciler struct {
 	client.Client
-	Scheme      *runtime.Scheme
-	Recorder    record.EventRecorder
-	RESTMapper  meta.RESTMapper
-	Config      *rest.Config
-	Dynamic     *dynamic.DynamicClient
-	InputGVK    schema.GroupVersionKind
-	InputGVR    schema.GroupVersionResource
-	Composition types.NamespacedName
+	Scheme            *runtime.Scheme
+	Recorder          record.EventRecorder
+	RESTMapper        meta.RESTMapper
+	Config            *rest.Config
+	Dynamic           *dynamic.DynamicClient
+	InputGVK          schema.GroupVersionKind
+	InputGVR          schema.GroupVersionResource
+	Composition       types.NamespacedName
+	CRDChangedWatcher chan event.GenericEvent
```

Comment:
Is this meant to indicate Composition change ? 
If so does this name `CompositionChangeWatcher` seems more appropriate ?

Reply:
Ahh, you're right. Must have gotten confused. Fixed.